packages/asc-ui/src/components/TopBar/StyledTopBar.ts

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+
+const defaultBackgroundColor = '#fff'
```

Comment:
Use color from the theme

Reply:
Preferably create an object containing the const defaultProps = { ... }

Reply:
done